e2e/editor-tabs.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form'
+  );
+});
+
+test('should toggle editor visibility based on aria-expanded attribute', async ({
+  page
+}) => {
+  const tabLocator = page.locator('div.monaco-editor-tabs button').first();
+  const isExpandedBeforeClick = await tabLocator.getAttribute('aria-expanded');
+  if (isExpandedBeforeClick === 'true') {
+    await tabLocator.click();
+    const isExpandedAfterClick = await tabLocator.getAttribute('aria-expanded');
+    expect(isExpandedAfterClick).toBe('false');
+  } else {
+    await tabLocator.click();
+    const isExpandedAfterClick = await tabLocator.getAttribute('aria-expanded');
+    expect(isExpandedAfterClick).toBe('true');
+  }
+});
+
+test('toggling the button should toggle the appearance of the corresponding ReflexElement', async ({
+  page
+}) => {
+  const tabsLocator = await page.locator('div.monaco-editor-tabs button').all();
+  for (const tab of tabsLocator) {
+    const tabFileKey = await tab.getAttribute('data-cy');
```

Comment:
I wrote the test in this way because there is a mapping between the two elements I want to test on 
```
<div data-cy="editor-container-indexhtml" >
```
```
<div data-cy="editor-tab-indexhtml">
```
Their visibilities are related.

I am not sure `getByRole` or `getByText` can suffice this purpose and if we don't want to use `data-cy` I can add another attribute to reflect this mapping but the logic is the same.